### ISSUE TITLE
Train/validation split

### ISSUE THREAD

**Issue by cloudfool** [xiaohaoChen/rrc_detection#31]
Hi,

Could you please share your train and validation set (pic ID) to me? 
Thanks.

**Comment by cloudfool** [comment]
solved

**Comment by foralliance** [comment]
@cloudfool 
how did u solved？？
many many thanks

**Comment by cloudfool** [comment]
@foralliance Please see data/KITTI-car/main

**Comment by foralliance** [comment]
@cloudfool 
about "data/KITTI-car/main",this division is not mentioned in the paper.
`The goal was tomake the training set as different from the validation set aspossible. Our resulting validation set has 2741 images.`
[](https://github.com/xiaohaoChen/rrc_detection/issues/14)a discussion has been made.

**Comment by foralliance** [comment]
about "data/KITTI-car/main",this division is not mentioned in the paper.
`The goal was tomake the training set as different from the validation set aspossible. Our resulting validation set has 2741 images.`
a discussion has been made：[](https://github.com/xiaohaoChen/rrc_detection/issues/14)

**Comment by foralliance** [comment]
about "data/KITTI-car/main",this division is not mentioned in the paper.
`The goal was tomake the training set as different from the validation set aspossible. Our resulting validation set has 2741 images.`
a discussion has been made：[https://github.com/xiaohaoChen/rrc_detection/issues/14](url)

**Comment by cloudfool** [comment]
The 2741 validation images can be seen in data/KITTI-val/main

**Comment by foralliance** [comment]
many many thanks
